Examples/Image/Detection/FastRCNN/A1_GenerateInputROIs.py

```diff
@@ -7,14 +7,15 @@
 from __future__ import print_function
 from builtins import input
 import os, sys, datetime
+import cv2
 import numpy as np
 import shutil, time
 import PARAMETERS
 
 from cntk_helpers import makeDirectory, getFilesInDirectory, imread, imWidth, imHeight, imWidthHeight,\
                          getSelectiveSearchRois, imArrayWidthHeight, getGridRois, filterRois, imArrayWidth,\
                          imArrayHeight, getCntkInputPaths, getCntkRoiCoordsLine, getCntkRoiLabelsLine, roiTransformPadScaleParams,\
-                         roiTransformPadScale, cntkPadInputs
+                         roiTransformPadScale, cntkPadInputs, imresizeMaxDim
```

Comment:
It is also being used [here](https://github.com/rotimpe/CNTK/blob/2f40b8a2e0d4c589051e844426f136a0d5d27635/Examples/Image/Detection/FastRCNN/A1_GenerateInputROIs.py#L64)